docs/synapse/userguides/storm_adv_control.rstorm

```diff
@@ -232,9 +179,8 @@ Fini Block
 ++++++++++
 
 A **fini block** allows you to execute the specified Storm **once** after all nodes have passed through
-the preceding code in the Storm pipeline. This allows you to use Storm to perform a set of operations
-a single time "outside" of the normal Storm pipeline; without the fini block, the code would be executed
-once for **each** node passing through the query.
+the Storm pipeline. This allows you to use Storm to perform a set of operations a single time at the end
```

Comment:
in the init block docs you have "single" bolded but not here

Reply:
added bold